sagemaker-pipelines/data-ingestion/feature-store-data-ingestion-via-sagemaker-model-building-pipeline.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>in the usages i see "Data Wrangler" should be two words</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>